Travlendar/frontend/src/components/eventsList/listevent.js

```diff
@@ -22,6 +22,8 @@ class ListEvent extends Component{
         eventService.getEvents().then(function (result) {
             console.log(result);
             self.setState({ events:  result.data})
+        }).catch(()=>{
```

Comment:
1. Modify listevent.css so that events will be displayed below Header properly.
2. Rename files listevent.js and listevent.css with coding standards (ListEvent.js, ListEvent.css)